src/loaders/IVolumeLoader.ts

```diff
@@ -35,7 +34,7 @@ export class VolumeDims {
  * @param {Volume} volume
  * @param {number} channelindex
  */
-export type PerChannelCallback = (imageurl: string, volume: Volume, channelIndex: number) => void;
+export type PerChannelCallback = (volume: Volume, channelIndex: number) => void;
```

Comment:
I think the reason the url is part of the signature here was so that the function implementation can make sure that the data is arriving for the intended target.  This was in part solving the problem of trying to load one volume and then requesting a totally different image while the first one was in flight.  Hopefully our request queue class will be the solution for this. It's definitely a scenario we need to be aware of.